source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs

```diff
@@ -12,12 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Energinet.DataHub.Wholesale.CalculationResults.Interfaces.CalculationResults;
+
 namespace Energinet.DataHub.Wholesale.Events.Application.UseCases;
 
 public class AggregatedTimeSeriesRequestHandler : IAggregatedTimeSeriesRequestHandler
 {
-    public async Task ProcessAsync(CancellationToken cancellationToken)
+    private readonly ICalculationResultQueries _calculationResultQueries;
+
+    public AggregatedTimeSeriesRequestHandler(ICalculationResultQueries calculationResultQueries)
+    {
+        _calculationResultQueries = calculationResultQueries;
+    }
+
+    public Task ProcessAsync(CancellationToken cancellationToken)
```

Comment:
nit: I anticipate that the handler receives a request. Making this explicit will make the signature more explicit and make it easier to reason about your design. First step could be like this:
```suggestion
    public Task ProcessAsync(object request, CancellationToken cancellationToken)
```

Reply:
Good point, will be added in next PR